src/cttvApi.js

```diff
@@ -43,7 +43,7 @@ var cttvApi = function () {
         if (format === "json") {
             req = jsonHttp;
         } else {
-            req = http;
+            req = jsonHttp;
```

Comment:
This change will always set the request and response headers to accept json, which is not what we want